packages/quantic/force-app/main/default/lwc/quanticRecentQueriesList/quanticRecentQueriesList.css

```diff
@@ -10,15 +10,20 @@
 }
 
 .recent-query__icon {
-  margin-right: .5rem;
-  margin-left: .25rem;
+  margin-right: 0.5rem;
+  margin-left: 0.25rem;
 }
 
 .empty-list-message {
-  color: var(--sds-c-color-gray-8, #8E959D);
+  color: var(--sds-c-color-gray-8, #8e959d);
 }
 
 li:hover {
   text-decoration: underline;
   cursor: pointer;
-}
\ No newline at end of file
+  text-decoration-color: var(--brand-text-link, #0176d3);
+}
+
+li div {
+  pointer-events: none;
```

Comment:
Why is this needed?

Reply:
IIRC without, this clicks on the icon or actual label would not cause the search to execute. (seems like it might relate to the issue at hand). It's possible it was a suggestion I made prior to some html changes that might cause it not to work anymore.

Reply:
so that the onClick of the text and the icon takes the onClick of the parent . In other words to avoid the onclicks of the child (divs)

Reply:
previously I was using event.target, for which the events weren't bubbling up but now on using event.currentTarget, the issue gets solved. I don't need to use pointer-events: none for child divs